UI/data/locale/en-US.ini

```diff
@@ -684,6 +693,10 @@ Basic.Main.GroupItems="Group Selected Items"
 Basic.Main.Ungroup="Ungroup"
 Basic.Main.GridMode="Grid Mode"
 Basic.Main.ListMode="List Mode"
+Basic.Main.LockPreview="Lock Preview"
+Basic.Main.UnlockPreview="Unlock Preview"
+Basic.Main.VirtualCam="Virtual Camera"
```

Comment:
I'm not sure this actually requires a text string. Maybe on hover, but not in the button itself, as it's not as critical of a flow.

Reply:
Agreed. Updated to remove text from the button.